app/features/idl/interactive-idl/model/use-instruction-form.ts

```diff
@@ -3,6 +3,49 @@ import { type Path, type SubmitHandler, useForm, type UseFormSetValue } from 're
 
 import { isRequiredArg } from '../lib/instruction-args';
 
+type ArgumentType =
+    | 'bool'
+    | 'u8'
+    | 'u16'
+    | 'u32'
+    | 'u64'
+    | 'u128'
+    | 'i8'
+    | 'i16'
+    | 'i32'
+    | 'i64'
+    | 'i128'
+    | 'f32'
+    | 'f64'
+    | 'string'
+    | 'bytes'
+    | 'publicKey'
+    | 'pubkey';
+
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+const DEFAULT_VALUES_PER_TYPE: Record<ArgumentType, string> = {
+    bool: 'false',
+    u8: '1',
+    u16: '1',
+    u32: '1',
+    u64: '1',
+    u128: '1',
+    i8: '1',
+    i16: '1',
+    i32: '1',
+    i64: '1',
+    i128: '1',
+    f32: '1.0',
+    f64: '1.0',
+    string: 'default',
+    bytes: 'data',
+    publicKey: '11111111111111111111111111111111',
+    pubkey: '11111111111111111111111111111111',
```

Comment:
I do not have better idea for this address, but if we keep default one, it is better to use PublicKey.default